ui/components/app/network-display/index.scss

```diff
@@ -1,3 +1,5 @@
+@use "sass:color";
```

Comment:
Is this still needed? I don't see a reference to it? Although I guess if it wasn't it should have been picked up by the linter 🤔

Reply:
Removed in [15a862a](https://github.com/MetaMask/metamask-extension/pull/13896/commits/15a862ad7ec17f97abc4586d79b7eaa95bae317b).

Reply:
Weird that the linter didn't pick that up haha